.travis.yml

```diff
@@ -66,6 +66,19 @@ matrix:
         - DESC="package no error"
         - CMD="mvn clean package -Passembly"
 
+    # checkstyle regression no exception (oraclejdk8)
+    - jdk: oraclejdk8
+      env:
+        - DESC="checkstyle regression no exception"
+        - CMD1="git clone https://github.com/checkstyle/checkstyle &&"
+        - CMD2="cd checkstyle &&"
+        - CMD3="git checkout -b test-branch &&"
+        - CMD4="git checkout master &&"
+        - CMD5="git reset --hard HEAD^ &&"
```

Comment:
What about testing more pairs: commit1(branch), commit2(master); commit2(branch), commit3(master); commit3(branch), commit4(master)...

> Can we add one more with specific commits which we know will produce a configuration?

That means we need to modify one of the module and make commit.